Apps/Common/SIMSolverAdap.h

```diff
@@ -45,6 +46,12 @@ template<class T1> class SIMSolverAdap : public SIMSolverStat<T1>
     if (!aSim.initAdaptor())
       return 1;
 
+    if (SIMSolverStat<T1>::exporter)
+      SIMSolverStat<T1>::exporter->setFieldValue(exporterName, &this->S1,
+                                                 &aSim.getSolution(),
+                                                 &aSim.getProjections(),
+                                                 &aSim.getEnorm());
```

Comment:
This commit (5e8215a4182d4fff0ccea3dbbd514200c72f873b) is too early, it seams, since the extension of the setFieldValue signature comes in a later commit. So this one should either be moved to after that one, or squashed into it.